app.js

```diff
@@ -48,6 +48,10 @@ app.get('/elite2api/health', (req, res) => {
   res.send({ status: 'UP' })
 })
 
+app.get('/probationteams/health/ping', (req, res) => {
+  res.send('pong')
```

Comment:
shouldn't that be `{ status: 'UP' }` too?

Reply:
not sure of the url for probation teams but have a look at https://keyworker-api-dev.prison.service.justice.gov.uk/health/ping

Reply:
yes it should be:  https://probation-teams-dev.prison.service.justice.gov.uk/health/ping.
It wont make any diff as it only relies on the status being 200 - but fixed to be consistent.